backtrace-library/src/main/java/backtraceio/library/BacktraceDatabase.java

```diff
@@ -175,6 +176,11 @@ public Boolean setupNativeIntegration(BacktraceBase client, BacktraceCredentials
         if (minidumpSubmissionUrl == null) {
             return false;
         }
+        
+        if (client.usesCustomRequestHandler()) {
+            minidumpSubmissionUrl = "";
```

Comment:
Yes.  Because if they are using a custom request handler we want to force Crashpad to not send and I tried setting this to a null but it causes a crash.